README.md

```diff
@@ -42,6 +42,10 @@ export PATH=$PATH:<CLI_HOME>
 ```
 
 3. Let's create our first project with name "petstore" by adding the [open API definition](https://petstore.swagger.io/v2/swagger.json) of the petstore . You can do that by executing the following command using your command line tool.
+
+  NOTE: If you have used a previous version of apictl before, remember to delete the directories
+ .wso2apictl and .wso2apictl.local that are located in `/home/<your-pc-username>`. Deleting them will make the newer apictl create them again, with content compatible with the current version.
+
```

Comment:
@SuKSW let's mention here to keep a backup and remove the directory.

Reply:
mentioning here to keep a backup and remove the directory.